x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/users/40_query.yml

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
Not sure what the number signifies here? Followed what looked like a convention to use +10 for the first test of every API.

Reply:
Regarding numbering, see this comment:
https://github.com/elastic/elasticsearch/pull/87691#discussion_r898729600

p.s. Using 40 is completely fine in this case.

Reply:
Thank you!

Reply:
lols my question exactly!